README.md

```diff
@@ -54,3 +54,51 @@ an email, file an issue, or ping me on IRC. Please note that I am lurking as
 zostay on irc.perl.org and Freenode, but it is unusual that I am actually
 looking at my chat window, so email is your best bet.
 
+## Getting Started
+
+* Install perl6 (For example, on Mac OSX, `brew install rakudo-star`
+  (rakudo is a compiler for Perl 6.  That command will put the `perl6`
+  executable in your path.  See [http://perl6.org/](http://perl6.org)
+  for more details or how to install on other platforms).
+
+* Clone this repository (e.g. `git clone git clone https://github.com/zostay/Smack.git` )
```

Comment:
> You duped "git clone" here, oops!

That's funny.  I do that on the command line pretty often, too, so it's
only fitting that I checked it in that way.  Thanks for catching that!

mike

On Thu, Jun 23, 2016 at 6:30 AM, Matt Creenan <notifications@github.com>
wrote:

> In README.md <https://github.com/zostay/Smack/pull/2#discussion_r68218765>
> :
>
> > @@ -54,3 +54,51 @@ an email, file an issue, or ping me on IRC. Please note that I am lurking as
> >  zostay on irc.perl.org and Freenode, but it is unusual that I am actually
> >  looking at my chat window, so email is your best bet.
> >
> > +## Getting Started
> > +
> > +* Install perl6 (For example, on Mac OSX, `brew install rakudo-star`
> > +  (rakudo is a compiler for Perl 6.  That command will put the `perl6`
> > +  executable in your path.  See [http://perl6.org/](http://perl6.org)
> > +  for more details or how to install on other platforms).
> > +
> > +* Clone this repository (e.g. `git clone git clone https://github.com/zostay/Smack.git` <https://github.com/zostay/Smack.git> )
>
> You duped "git clone" here, oops!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/zostay/Smack/pull/2/files/3bccb436d2044183433914370881bd2cb8f176e5#r68218765>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AB6E-1pqX5DVF-H3kr5Qu_K4_d4IE_ofks5qOm5tgaJpZM4I8X6F>
> .
>